2022-08-10 10:33:08.970289: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-08-10 10:33:08.970538: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Random seed: 8278653274190013290
---------------- Starting Your Algo --------------------
Configuring your custom algo strategy...
Could not spawn EI at location [24, 10]. Not enough resources.
Performing turn 0 of your custom algo strategy
Traceback (most recent call last):
  File "/home/wittmann/Desktop/c1terminal-algo/python-algo-1/algo_strategy.py", line 233, in <module>
    algo.start()
  File "/home/wittmann/Desktop/c1terminal-algo/python-algo-1/gamelib/algocore.py", line 74, in start
    self.on_turn(game_state_string)
  File "/home/wittmann/Desktop/c1terminal-algo/python-algo-1/algo_strategy.py", line 63, in on_turn
    game_state.submit_turn()
  File "/home/wittmann/Desktop/c1terminal-algo/python-algo-1/gamelib/game_state.py", line 180, in submit_turn
    send_command(build_string)
  File "/home/wittmann/Desktop/c1terminal-algo/python-algo-1/gamelib/util.py", line 29, in send_command
    sys.stdout.write(cmd.strip() + "\n")
BrokenPipeError: [Errno 32] Broken pipe
